source/extensions/filters/network/thrift_proxy/router/upstream_request.h

```diff
@@ -78,6 +78,8 @@ struct UpstreamRequest : public Tcp::ConnectionPool::Callbacks,
   bool charged_response_timing_{false};
   MonotonicTime downstream_request_complete_time_;
   uint64_t response_size_{};
+
+  bool keep_downstream_;
```

Comment:
probably `close_downstream_on_error_` per the previous comments.

Reply:
Done